### PR TITLE
feat: fix tiptap, attachments overflow and stuff

### LINKED ISSUE #293: web: tiptap editor cant edit previous lines
when writing a message, you cant go back to edit previous lines
https://discord.com/channels/1113119653246545961/1227195951119073301

### PR DESCRIPTION
**Pull request by BlankParticle**
## What does this PR do?

Fixes #293

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] Chore (refactoring code, technical debt, workflow improvements)
- [ ] Enhancement (small improvements)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### Required

- [ ] Read [Contributing Guide](https://github.com/un/inbox/blob/main/CONTRIBUTING.md)
- [ ] Self-reviewed my own code
- [ ] Tested my code in a local environment
- [ ] Commented on my code in hard-to-understand areas
- [ ] Checked for warnings, there are none
- [ ] Removed all `console.logs`
- [ ] Merged the latest changes from main onto my branch with `git pull origin main`
- [ ] My changes don't cause any responsiveness issues

### Appreciated

- [ ] If a UI change was made: Added a screen recording or screenshots to this PR
- [ ] Updated the UnInbox Docs if changes were necessary

### CODE REVIEW

File: apps/web-app/pages/[orgShortcode]/convo/[id].vue [hunk 1]
```diff
@@ -454,7 +454,7 @@
                 :org-shortcode="orgShortcode">
                 <template #default="{ openFileDialog, loading }">
                   <UnUiButton
-                    label="Upload more"
+                    label=""
```
Comment (McPizza0) [inline apps/web-app/pages/[orgShortcode]/convo/[id].vue:457]: why label removed?
if we want just icon, should use square on the component

File: apps/web-app/pages/[orgShortcode]/convo/new.vue [hunk 2]
```diff
@@ -726,9 +726,10 @@
         :schema="z.string().trim().min(1).max(128)" />
     </div>
     <span class="text-sm font-medium">Message</span>
+
     <UnEditor
       v-model:modelValue="messageEditorData"
-      class="min-h-[150px] overflow-hidden" />
+      class="min-h-full overflow-hidden" />
```
Comment (McPizza0) [inline apps/web-app/pages/[orgShortcode]/convo/new.vue:732]: did you test this?
we dont want a full height editor if theres no messages

File: apps/web-app/pages/[orgShortcode]/convo/[id].vue (hunk 1 above)
Comment (BlankParticle) [reply apps/web-app/pages/[orgShortcode]/convo/[id].vue:457]: Yeah, I want a icon only

File: apps/web-app/pages/[orgShortcode]/convo/new.vue (hunk 2 above)
Comment (BlankParticle) [reply apps/web-app/pages/[orgShortcode]/convo/new.vue:732]: It works full height in new message only, in reply UI it is min height
I have tested it already, 150px was not covering the whole Message Box

File: apps/web-app/pages/[orgShortcode]/convo/new.vue [hunk 3]
```diff
@@ -761,7 +762,7 @@
         :org-shortcode="orgShortcode">
         <template #default="{ openFileDialog, loading }">
           <UnUiButton
-            label="Upload more"
+            label=""
```
[the remark this answers is not in the document]
Comment (BlankParticle) [reply apps/web-app/pages/[orgShortcode]/convo/new.vue:765]: I didn't knew about it, get it fixed

File: apps/web-app/pages/[orgShortcode]/convo/[id].vue (hunk 1 above)
Comment (McPizza0) [reply apps/web-app/pages/[orgShortcode]/convo/[id].vue:457]: ok, remove label=""
add attribute `square`

**Review by McPizza0 — DISMISSED** [review]

**Review by McPizza0 — APPROVED** [review]

**Pull request merged**